### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
[DEMO LINK](https://Zelinskyi-Serhii.github.io/react_tabs/)

### CODE REVIEW

**Review by milenadiamonds1 — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -1,42 +1,44 @@
...
+  const getTitle = () => {
+    const findTab = tabs.find(tab => tab.id === selectedTabId);
 
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
+    return findTab
+      ? findTab.title
+      : tabs[0].id;
+  };
```
Comment (OleksiiBlezniuk) [inline src/App.tsx:23]: I don't think that this function is required here, let's just save tab to constant

File: src/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -1 +1,63 @@
...
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  onTabSelected,
+  selectedTabId,
+}) => {
+  const getTab = () => {
+    const findTab = tabs.find(tab => tab.id === selectedTabId);
+
+    return findTab || tabs[0];
+  };
```
Comment (OleksiiBlezniuk) [inline src/components/Tabs/Tabs.tsx:22]: Just save it to constant

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]

File: src/components/Tabs/Tabs.tsx [hunk 3]
```diff
@@ -1 +1,61 @@
...
+  onTabSelected: FuncSelected,
+  selectedTabId: string,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  onTabSelected,
+  selectedTabId,
+}) => {
+  const findTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```
Comment (illiarerun) [inline src/components/Tabs/Tabs.tsx:19]: you're doing it 2 times, create a helper function for it

File: src/components/Tabs/Tabs.tsx [hunk 4]
```diff
@@ -1 +1,61 @@
...
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const isSelected = selectedTabId === tab.id;
+
+            return (
+              <li
+                data-cy="Tab"
+                className={classNames({
+                  'is-active': tab.id === findTab.id,
```
Comment (illiarerun) [inline src/components/Tabs/Tabs.tsx:32]: use `isSelected` here

**Review by illiarerun — CHANGES_REQUESTED** [review]

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**